### PR TITLE
[2.13.x] DDF-04900 Improve policy file documentation

### PR DESCRIPTION
**Pull request by ahoffer**
#### What does this PR do?
 Adds detail to documentation in policy configuration fie

#### Who is reviewing it? 
@emmberk @brjeter @austinsteffes @ricklarsen @tyler30clemens  @peterhuffer 

#### Select relevant component teams: 
@codice/docs 

#### Ask 2 committers to review/merge the PR and tag them here.
@brjeter
@clockard
@emmberk
@mcalcote

#### How should this be tested?
N/A

#### Any background context you want to provide?
No.

#### What are the relevant tickets?
Fixes: #4900 

#### Checklist:
- [X] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: distribution/ddf-common/src/main/resources/security/configurations.policy [hunk 1]
```diff
@@ -1,80 +1,133 @@
...
-//
-// If you want to change the default location of the backup security log files, add new permissions
-// allowing the logger to read and modify the files within the new directory. If you haven't
-// already done so, follow the Enabling Fallback Audit Logging steps in the security hardening
-// checklist to finish configuring backup logging.
+
+
+// ****************************  BEGIN BACKUP LOG FILE SECTION  ************************************
+// * If you want to change the default location of the backup security log files, add new permissions
+// allowing the logger to read and modify the files within the desired directory for log files.
```
Comment (emmberk) [inline distribution/ddf-common/src/main/resources/security/configurations.policy]: ✏️ add a space before `allowing` to be consistent with other sections

File: distribution/ddf-common/src/main/resources/security/configurations.policy [hunk 2]
```diff
@@ -1,80 +1,133 @@
 priority "grant";
-// Double forward slashes "//" indicate comments and have no effect on policies.
+
+// This file allows administrators to add permissions
+// necessary for their particular configuration. This file is organized into sections, one
```
Comment (emmberk) [inline distribution/ddf-common/src/main/resources/security/configurations.policy]: ✏️ `sections: one`

File: distribution/ddf-common/src/main/resources/security/configurations.policy [hunk 3]
```diff
@@ -1,80 +1,133 @@
...
+// *************************************************************************************************
+
+
+
+// ***********************  BEGIN CONTENT DIRECTORY MONITOR SECTION  *******************************
+// * Configuring the CDM requires adding read and write permissions to the directory being monitored.
+// * If monitoring a WebDav server, then adding these permissions is not required and
+//   these instructions can be ignored.
+// * If a CDM configuration is deleted, then the corresponding permissions that were added should be
+//   removed from this file to avoid granting unnecessary permissions to parts of the system.
```
Comment (emmberk) [inline distribution/ddf-common/src/main/resources/security/configurations.policy]: ✏️ Might be a good idea to add a note here that the URL resource reader also needs permissions to read from the directory if the CDM is in-place.

**Comment by ahoffer** [comment]
@emmberk When the changes to the policy file get merged, I'll open a new Github issue and see what I can do about the docs.

**Pull request merged**